tools/mo/openvino/tools/mo/front/tf/InterpolateTransposes.py

```diff
@@ -58,6 +58,9 @@ def replace_sub_graph(self, graph: Graph, match: dict):
         transpose_1 = match['transpose_1']
         transpose_2 = match['transpose_2']
 
+        if interpolate.in_port(0).data.get_shape() is None:
+            return # get_axes for opset4::Interpolate only works if the input shape is known
```

Comment:
You seem to call `get_shape()` on the front phase before partial shape inference. It looks strange to me. Please double-check